index.bs

```diff
@@ -8501,8 +8488,25 @@ as discussed in [[#sctn-username-enumeration]].
 
 [=[RPS]=], at [=registration=] time, SHOULD provide affordances for users to complete future [=authorization gestures=] correctly. This could involve naming the authenticator, choosing a picture to associate with the device, or entering freeform text instructions (e.g., as a reminder-to-self).
 
+
+## Recommended Ranges for Ceremony Timeouts ## {#sctn-timeout-recommended-ranges}
+
 [=Ceremonies=] relying on timing, e.g., a [=registration ceremony=] (see {{PublicKeyCredentialCreationOptions/timeout}}) or an [=authentication ceremony=] (see {{PublicKeyCredentialRequestOptions/timeout}}), ought to follow [[!WCAG21]]'s [Guideline 2.2 Enough Time](https://www.w3.org/TR/WCAG21/#enough-time). If a [=client platform=] determines that a [=[RP]=]-supplied timeout does not appropriately adhere to the latter [[!WCAG21]] guidelines, then the [=client platform=] MAY adjust the timeout accordingly.
 
+The <dfn>recommended ranges and defaults for a WebAuthn ceremony timeout</dfn> are as follows:
+
+If the [=[RP]=]'s [=user verification=] preference for the ceremony is
+
+  <dl class="switch">
```

Comment:
See also https://github.com/w3c/webauthn/pull/1885 where I am proposing adjusting these ranges.

Reply:
Updated to account for the changes in #1885.